roles/neutron-data-network/tasks/main.yml

```diff
@@ -262,6 +278,24 @@
         - /etc/neutron/services/loadbalancer/haproxy/lbaas_agent.ini
         - /etc/neutron/neutron_lbaas.conf
 
+- name: install neutron lbaas service (rhel distro)
+  systemd_service:
```

Comment:
are you sure neutronlbaas package doesn't provide a systemd unit already?  most of the rhel packages do.